pandunia-lekse.csv

```diff
@@ -1599,7 +1606,7 @@ konkara||||collaborate (work together)|collaborer|colaborar|colaborar|сотру
 kono|||zho:共 (gòng), yue:共 (gung4), kor:공 (gong), vie:cộng, jpn:共 (kyō-), eng:fra:spa:por:con-|together (jointly, mutually)||juntos||||共同 (一起)|一緒に||||||pamoja|birlikte|kune|yhdessä|
 konserte|||eng:fra:concert, spa:concierto, por:concerto, deu:Konzert, rus:концерт (koncert), may:tur:konser, jpn:コンサート (konsāto), kor:콘서트 (konseoteu), hin:कॉन्सर्ट (kŏnsarṭ)|concert|concert|concierto|concerto|концерт||音乐会|コンサート (音楽会)|콘서트 (음악회)||कॉन्सर्ट|সঙ্গীতানুষ্ঠান|konser||konser|koncerto|konsertti|koncert
 konsida|||eng:consider, spa:considerar|consider (regard)||considerar (pensar en)|||||||||||||rigardi ia|pitää jonakin|rozważać, brać pod uwagę
-konsone|||fra:consonne, eng:consonant|consonant||consonante||||||||||||||konsonantti|spółgłoska
+konsonde|||fra:consonne, eng:consonant|consonant||consonante||||||||||||||konsonantti|spółgłoska
```

Comment:
Both konsone (together-sleep) and konsonde (together-sound) are unsuitable. Maybe vowel could be core-sound and consonant could be something like border-sound because those are their roles in syllables?

Reply:
In Romance languages and Armenian consonants are "together-sounds" and vowels are "voice-sounds", while in Slavic and Polynesian languages they are "together-sounds" and "lone-sounds", so that was the aim here. It does seem to be an outdated description now that syllabic consonants are widely known to linguists, though. core- and border- sound good.